cli/src/pcluster/validators/cluster_validators.py

```diff
@@ -601,7 +601,7 @@ def _validate(self, shared_storage_name_mount_dir_tuple_list, local_mount_dir_in
                     f"clashes with the one used for ephemeral volumes of the instances {list(instance_types)}. "
                     f"Please change the mount directory configuration of either the shared storage or the ephemeral "
                     f"volume of the impacted nodes.",
-                    FailureLevel.ERROR,
+                    FailureLevel.WARNING,
```

Comment:
Did you verify that no corresponding changes in unit tests are required?

Reply:
No unit test change

Reply:
So are we missing to test it or are we just skipping to test the FailureLevel?